### PR TITLE
Created file uploading endpoint

### PR DESCRIPTION
**Pull request by jordan722**

### CODE REVIEW

File: api/app.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import express from "express";
 import bodyparser from "body-parser";
 import logger from "morgan";
+
+const multer = require('multer');
```
Comment (kionkina) [inline api/app.ts:5]: should we do something like import * as multer from 'multer' for consistency?
was require giving you an error earlier?

**Review by kionkina — APPROVED** [review]

File: api/app.ts (hunk 1 above)
Comment (boubascript) [reply api/app.ts:5]: `import multer from "multer";` should work

Comment (jordan722) [reply api/app.ts:5]: The import statement gives me errors so I stuck with require

Comment (boubascript) [reply api/app.ts:5]: oh you have to `npm install @types/multer` thats whats missing and should solve it.

**Pull request merged**